Game/MyProject/IvGraphics/OGL/IvRendererOGL.cpp

```diff
@@ -198,11 +198,6 @@ IvRendererOGL::Resize(unsigned int width, unsigned int height )
 
     SetProjectionMatrix(perspective);
```

Comment:
When the app resizes it will no longer use your camera's projection matrix, rather the one defined in this function.
You can remove this as well, just do viewport resizing.